### PR TITLE
Changed enum from environment to global

### PR DESCRIPTION
**Pull request by manikanta-reddy-t**

### CODE REVIEW

File: automator/src/com/testsigma/automator/entity/TestDataType.java [hunk 1]
```diff
@@ -32,6 +32,8 @@ public static TestDataType getTestDataType(String name) {
         return random;
       case "function":
         return function;
+      case "global":
```
Comment (tarun-testsigma) [inline automator/src/com/testsigma/automator/entity/TestDataType.java:35]: case environment also has to be removed

File: server/src/main/java/com/testsigma/model/TestDataType.java [hunk 2]
```diff
@@ -15,7 +15,7 @@ public enum TestDataType {
   raw(1, "raw"),
   parameter(2, "parameter"),
   runtime(3, "runtime"),
-  environment(4, "environment"),
+  global(4, "global"),
```
Comment (tarun-testsigma) [inline server/src/main/java/com/testsigma/model/TestDataType.java:18]: @manikanta-reddy-t You also need to migrate the existing data in test step table from environment to global

File: automator/src/com/testsigma/automator/entity/TestDataType.java (hunk 1 above)
Comment (manikanta-reddy-t) [reply automator/src/com/testsigma/automator/entity/TestDataType.java:33]: done

File: server/src/main/java/com/testsigma/model/TestDataType.java (hunk 2 above)
Comment (manikanta-reddy-t) [reply server/src/main/java/com/testsigma/model/TestDataType.java:18]: written migration query for this

**Review by tarun-testsigma — APPROVED** [review]

**Review by rajesh-testsigma — COMMENTED** [review]
bootstrap SQL changes missing. Have you added the change in automator and agent ?

**Pull request merged**